runecoral/runecoral.cpp

```diff
@@ -17,6 +17,7 @@ const char* RUNE_CORAL_MIME_TYPE__TFLITE = "application/tflite-model";
 
 RuneCoralTensor to_runecoraltensor(const TfLiteTensor &tfLiteTensor) {
     RuneCoralTensor result;
+    result.name = tfLiteTensor.name;
```

Comment:
Is there any chance the `TfLiteTensor`'s name could move or be deallocated before the overall inference context is destroyed?

Reply:
Nope. because the interpreter owns the tensors..  and InferenceContext owns the interpreter..